config/routes.rb

```diff
@@ -12,7 +12,7 @@
     get 'vmlist'
   end
 
-  resource :admin do
+  resource :admin, only: :show do
```

Comment:
@icanberk This line suffices because having `resource :admin` implies the full RESTful routes. Meanwhile, limiting it with `only: :show` means I'm excluding the routes for `create`, `edit`, and `delete`.

You can also run a routes summary and check the differences.